docs/proposals/20191030-machine-health-checking.md

```diff
@@ -108,6 +105,15 @@ As an operator of a Management Cluster, I want my machines to be self-healing an
 
 ### Implementation Details/Notes/Constraints
 
+#### Machine annotation:
+```go
+const MachineUnhealthyAnnotation = "machine.cluster.x-k8s.io/unhealthy"
+```
+
+This annotation is applied by the MHC controller and it is expected that it
+will never be removed. The only current mediation strategy is deletion, which
+is to be handled by the machines owning controller.
```

Comment:
nit: took me a moment to interpret this, could also be clearer maybe if it were `is to be handled the controller owning the machine.`
```suggestion
is to be handled by the machine's owning controller.
```